fuels-signers/Cargo.toml

```
@@ -25,3 +27,4 @@ tokio = { version = "1.10.1", features = ["full"] }
 
```

If you add your own test-helpers flag, you can conditionally enable fuel-core's test-helpers based on your flag:

```suggestion
[features]
test-helpers = ["fuel-core/test-helpers"]
```